source/getting-started/tutorials/sensors/index.md

```diff
@@ -274,10 +273,4 @@ Run Min:	3.25 usec
 Run Max:	3.84 usec
 Run Mean:	3.39 usec
 Run Var:	0.00 usec
-```
-
-## TODO
-
-ISR Generation mode: Legacy vs New Mode
```

Comment:
I think it is good to remove this, so please keep it removed. However, I have always thought it a bit strange that we went through all the work to redesign the system architecture around the Timing Mananger, and then the default configuration in `user_config.h` is for the Legacy Mode, and not the Post-Sensor Mode.

Nathan recommended this choice to maintain backwards-compatibility, but if users are following this tutorial, we should recommend at the beginning of this tutorial that they switch the configuration from Legacy Mode to Post-Sensor Mode in `user_config.h`. Please add instructions for that to the beginning of this tutorial.

Reply:
There are instructions for changing to post-sensor mode, it's the first thing in the C-Code section